backend/persistent/PlotlyAPI.js

```diff
@@ -26,15 +26,34 @@ export function PlotlyAPIRequest(relativeUrl, {body, username, apiKey, accessTok
     });
 }
 
-export function newDatacache(payloadJSON, type) {
+export function newDatacache(payloadJSON, type, requestor) {
```

Comment:
no need for `requestor` since this PR already adds it to `payloadJSON`

Reply:
Thanks !! Will update it..

Reply:
Update: 

https://github.com/plotly/falcon-sql-client/pull/313#discussion_r156592084 is valid for here as well, so this `requestor` is needed.